yarn.lock

```diff
@@ -1401,6 +1401,18 @@ etag@~1.8.1:
   version "1.8.1"
   resolved "https://registry.yarnpkg.com/etag/-/etag-1.8.1.tgz#41ae2eeb65efa62268aebfea83ac7d79299b0887"
 
+event-stream@^4.0.1:
+  version "4.0.1"
+  resolved "https://registry.yarnpkg.com/event-stream/-/event-stream-4.0.1.tgz#4092808ec995d0dd75ea4580c1df6a74db2cde65"
+  dependencies:
+    duplexer "^0.1.1"
+    from "^0.1.7"
+    map-stream "0.0.7"
+    pause-stream "^0.0.11"
+    split "^1.0.1"
+    stream-combiner "^0.2.2"
+    through "^2.3.8"
+
 event-stream@~3.3.0:
```

Comment:
The old version of event-stream is still here, just adding a top-level dependency in NPM doesn't mean that it will be used by other dependencies, since multiple versions can be in play at the same time.

I think you'll need to use the `resolutions` feature: https://yarnpkg.com/lang/en/docs/selective-version-resolutions/